exadata-database-service-on-cc/introduction/introduction.md

```diff
@@ -0,0 +1,81 @@
+# Introduction
+
+The labs in this workshop walk you through all the steps to get started using the Oracle Exadata Database Service to perform lifecycle tasks and procedures.
+
```

Comment:
@leoneila Add Estimated Workshop Time: the time to complete the entire workshop

Reply:
Added estimated workshop